hazelcast/src/main/java/com/hazelcast/map/impl/operation/MultipleEntryBackupOperation.java

```diff
@@ -70,7 +71,7 @@ protected void readInternal(ObjectDataInput in) throws IOException {
         super.readInternal(in);
         backupProcessor = in.readObject();
         int size = in.readInt();
-        keys = new HashSet<Data>(size);
+        keys = new LinkedHashSet<Data>(size);
```

Comment:
We need to iterate keys in the order that they are read to keep wan events ordered.